indexer/scales_patch.hpp

```diff
@@ -15,7 +15,7 @@ inline uint32_t PatchMinDrawableScale(uint32_t s)
 
 inline uint32_t PatchMaxDrawableScale(uint32_t s)
 {
-  std::min(s + kPatchScaleShift, static_cast<uint32_t>(GetUpperStyleScale()));
+  return std::min(s + kPatchScaleShift, static_cast<uint32_t>(GetUpperStyleScale()));
```

Comment:
Should  this fix go to the master separately?

Reply:
It's used in the search and can be merged to master independently. Isn't it UB? @vng

Reply:
It's covered with `#ifdef BUILD_DESIGNER`